src/lib/constants.js

```diff
@@ -0,0 +1,13 @@
+export const ERROR_MESSAGE_DEFAULT = '[ERROR]';
+export const ERROR_MESSAGE_USER_INPUT = `${ERROR_MESSAGE_DEFAULT} 입력 도중 알 수 없는 에러가 발생했습니다.`;
+export const ERROR_MESSAGE_CAR_NAME_INVALID = `${ERROR_MESSAGE_DEFAULT} 1글자 이상, 5글자 이하의 알파벳, 숫자 혹은 언더바(_)로 이루어진 자동차 이름을 입력해주세요`;
+export const ERROR_MESSAGE_CAR_NAME_DUPLICATION = `${ERROR_MESSAGE_DEFAULT} 중복된 자동차 이름을 입력할 수 없습니다.`;
+export const ERROR_MESSAGE_NOT_INTEGER = `${ERROR_MESSAGE_DEFAULT} 정수를 입력해주세요.`;
+export const ERROR_MESSAGE_NOT_POSITIVE_POSITIVE = `${ERROR_MESSAGE_DEFAULT} 양수를 입력해주세요.`;
```

Comment:
이친구는 이름이 `NOT_POSITIVE_POSITIVE`인 이유가 있을까요?
```suggestion
export const ERROR_MESSAGE_NOT_POSITIVE = `${ERROR_MESSAGE_DEFAULT} 양수를 입력해주세요.`;
```

Reply:
허걱,, 오타인 것 같아요. 감사합니다.